src/components/SEO/SEO.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+
+import seoData from './seo-data.json';
+
+export const SEO = () => (
+  <Helmet>
+    <title>{seoData.title}</title>
```

Comment:
Have you checked what's going on with `google-site-verification` meta tag and the code wrapped with the `script` tag?

Reply:
We had a chat with Marcin and decided that
- the code wrapped in the `script` is a bit old and we don't need it at all
- the `google-site-verification` can be added later, once confirmed it's needed
- there _probably_ won't be any action required for the hotjar since we have a plugin for it (but if it turned out we need it on the landing page, this can be added later)
- we (devs) don't bother with the google search console

Reply:
Sounds great! Thanks!